### PR TITLE
Remove Outermost Parantheses #955

### PR DESCRIPTION
**Pull request by salonishah01**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Remove Outermost Parantheses #955

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#955

### CODE REVIEW

File: Python/other/remove_outermost_parenthesis.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            else:
+                start_bracket -= 1
+            if start_bracket == 0:
+                result += temp[1:-1]
+                temp = ""
+        return result
+
+str = "(()())(())"
+result = removeOuterParentheses(str)
+print(result)
```
Comment (iamrajiv) [inline Python/other/remove_outermost_parenthesis.py]: @salonishah01 Add sample input-output, time and space complexity. Take reference from programs of Python section.

File: README.md [hunk 2]
```diff
@@ -40,6 +40,8 @@ Make sure your code works before submitting it :D
 
 **PLEASE READ THIS**
 
+LINK TO THE CODE - https://github.com/salonishah01/NeoAlgo/blob/outermost-paranthesis/Python/other/remove_outermost_parenthesis.py
```
Comment (iamrajiv) [inline README.md]: @salonishah01 Do not update like this. Take reference to how other programs are added.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@salonishah01 Remove unecessay files i.e. `CircularLinkedList.c`.

**Comment by salonishah01** [comment]
@iamrajiv Above mentioned changes are made.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@salonishah01 You have updated the wrong README file. Add the link of the program in `NeoAlgo/Python/README.md`.

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@salonishah01 Do not make changes to `NeoAlgo/README.md`.

**Comment by salonishah01** [comment]
@iamrajiv I made changes to python/Readme.md

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@salonishah01 Do not make any changes to `NeoAlgo/README.md`. You are adding a line break to `NeoAlgo/README.md` file. Also, In future make your branch even with the main repository branch than make the PR since I can see that your branch is 806 commits behind `TesseractCoding:master`.

**Review by iamrajiv — APPROVED** [review]
@salonishah01 LGTM!

File: Python/README.md [hunk 3]
```diff
@@ -9,6 +9,101 @@
 
 # Codes in the Python language
 
+## Data Structures
```
Comment (atarax665) [inline Python/README.md:12]: Revert all the changes made to the README.md file. Just add a link of your file. Do not make changes to any of the existing links.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Code looks fine, make changes to the README file ASAP to get this PR merged.

File: Python/README.md (hunk 3 above)
Comment (salonishah01) [reply Python/README.md:12]: @atarax665 Done.

**Pull request closed without merging**